test/parse.js

```diff
@@ -31,4 +31,22 @@ describe( 'BlockMap.parse()', function() {
     })
   })
 
+  context( 'when file checksum is invalid', function() {
+
+    it( 'throws on invalid checksum', function() {
+      var json = require( `./data/version-2.0` )
+      var xml = fs.readFileSync( path.join( __dirname, `/data/invalid-file-checksum.bmap` ) )
+      assert.throws( function() {
```

Comment:
This will pass if anything throws, which might accidentally hide things like syntax errors, or calling undefined functions, etc. Maybe worth checking that a verification error was thrown?